warmup.py

```diff
@@ -240,17 +251,39 @@ def accumulate(self, L, points):
         The sample is accumulated into the radial distribution average.
 
         """
-        pass
+        if self.rmax >= L/2:
+            raise ValueError('rmax >= L/2')
+        temp_rdf = np.zeros_like(self._rdf)
+        for i, point in enumerate(points[:-1]):
+            neighbors = np.array(points[i+1:])
+            for neighbor in neighbors:
+                dist_vect = np.absolute(point - neighbor)
+                dist_vect -= L*np.round(dist_vect/L) 
+                dist_mag = np.linalg.norm(dist_vect)
+                if self.rmin <= dist_mag <= self.rmax:
+                    for j, bin_dist in enumerate(self.centers):
+                        r_outer = self.edges[j+1]
+                        r_inner = self.edges[j]
+                        if r_inner < dist_mag <= (r_outer + 1e-8):
+                            temp_rdf[j] += 2
```

Comment:
This works to find the right bin, but it is inefficient because it is what's called a "linear search". Every pair needs to loop until it finds the right distance, and this can take `len(self.centers)` iterations in the worst case scenario. Because we know that all of the bins are equally spaced, you can actually compute the bin.

To do this, first take the pair distance, and find its "fractional" bin coordinate in the binning range. For example, if the bin spacing is 0.5 and the binning range runs from [0,2], then the fractional coordinate for 1.25 would be 2.5. Then, you want to throw away the fractional part to get the bin because each bin covers an integer range, e.g., bin 2 covers [2,3). You can use the floor operation to do this.